issue_template.md

```diff
@@ -33,32 +45,36 @@ Confirm each of the following by checking the box.  This package:
 - [ ] includes documentation with examples for all functions.
 - [ ] contains a vignette with examples of its essential functions and uses.
 - [ ] has a test suite.
-- [ ] has continuous integration, including reporting of test coverage, using services such as Travis CI, Coveralls and/or CodeCov.
-- [ ] I agree to abide by [ROpenSci's Code of Conduct](https://github.com/ropensci/onboarding/blob/master/policies.md#code-of-conduct) during the review process and in maintaining my package should it be accepted.
+- [ ] has continuous integration, including reporting of test coverage using services such as Travis CI, Coveralls and/or CodeCov.
 
-#### Publication options
+## Publication options
 
 - [ ] Do you intend for this package to go on CRAN?  
 - [ ] Do you wish to automatically submit to the [Journal of Open Source Software](http://joss.theoj.org/)? If so:
```

Comment:
is it actuall automatic? i thought they had to submit manually to JOSS?

Reply:
I didn't change any of the text from our existing template. I just reformatted it. This is what our current template says. Example: